javascript/ql/src/semmle/javascript/frameworks/DateFunctions.qll

```diff
@@ -0,0 +1,61 @@
+private import javascript
+
+private API::Node formatFunction() {
+  result = API::moduleImport(["date-fns", "date-fns/utc"]).getMember(["format", "lightFormat"])
+  or
+  result =
+    API::moduleImport(["date-fns/format", "date-fns/lightFormat", "date-fns/utc/format",
+          "date-fns/utc/lightFormat"])
+}
+
+private API::Node formatFunctionCurried() {
+  result =
+    API::moduleImport(["date-fns/fp", "date-fns/fp/utc"]).getMember(["format", "lightFormat"])
+  or
+  result =
+    API::moduleImport(["date-fns/fp/format", "date-fns/fp/lightFormat", "date-fns/fp/utc/format",
+          "date-fns/fp/utc/lightFormat"])
+}
```

Comment:
Looks like you missed the `date-fns/esm` folder. ([Documentation](https://date-fns.org/v2.0.0-alpha.7/docs/ECMAScript-Modules)) 

The `esm` folder also contains a `esm/fp` sub folder.

Reply:
Good catch, After a closer look it seems the `utc` folder doesn't actually exist in the latest version of date-fns 1.x nor in 2.x, so I removed that at the same time.